vpc-endpoints.tf

```diff
@@ -185,6 +187,22 @@ resource "aws_vpc_endpoint" "sqs" {
   tags                = local.vpce_tags
 }
 
+#########################
+# VPC Endpoint for Lambda
+#########################
+resource "aws_vpc_endpoint" "lambda" {
+  count = var.create_vpc && var.enable_lambda_endpoint ? 1 : 0
+
+  vpc_id            = local.vpc_id
+  service_name      = format("com.amazonaws.%s.lambda", data.aws_region.current.name)
```

Comment:
Is there any reason for not using `aws_vpc_endpoint_service` data source to do exactly as most of other VPC endpoints?

If not, please update the code to be as in all other places.

Reply:
This has been updated to use the data source as indicated.